app/models/userModel.js

```diff
@@ -11,9 +11,8 @@ module.exports = (sequelize, DataTypes) => {
       },
 
       name: { type: DataTypes.STRING },
-      surname: { type: DataTypes.STRING, allowNull: false },
-      email: { type: DataTypes.STRING, allowNull: false, unique: true },
-      surname: { type: DataTypes.STRING, allowNull: false }
+      last_name: { type: DataTypes.STRING, allowNull: false },
```

Comment:
You could use an alias here to avoid using snake_case inside the code (where we have to use camelCase):
```js
lastName: { type: DataTypes.STRING, allowNull: false , field: 'last_name' },
```

Reply:
It is only valid for models (not migrations) since it is a code alias